helix-term/build.rs

```diff
@@ -13,5 +14,8 @@ fn main() {
         None => env!("CARGO_PKG_VERSION").into(),
     };
 
+    fetch_grammars().ok();
+    build_grammars().ok();
```

Comment:
I'd like to discard the failures to fetch and build because now you do the fetching/building after helix is compiled if you need. And this works well for nix, which will fail to fetch because of the sandbox. Are there failure modes that will end up just being confusing here? I think this would work particularly well with some additions to #1669 so it's very easy to troubleshoot: are grammars fetched, are grammars compiled.

Reply:
Hmm, on one hand, yeah, the helix binary doesn't necessarily need the grammars to compile, so I see your point. However, if we don't fail loudly, then we won't know if there is a problem with one or more of the grammars. The CI won't signal a problem, we'd have to go looking for it, or wait for a bug report.

Is there a way to make building and failing loudly the default, but have the ability to turn it off, like with an environment variable?

Reply:
Ah good idea, it probably makes sense to fail loudly as the default but be able to disable the auto fetching and building behaviors by setting an env var :+1: